.storybook/preview.js

```diff
@@ -1,4 +1,11 @@
 
 export const parameters = {
   actions: { argTypesRegex: "^on[A-Z].*" },
-}
\ No newline at end of file
+  html: {
+    prettier: {
```

Comment:
Nice stuff! I thought we're getting rid of prettier though? Or you want to keep it for html?

Reply:
That is just the plugin config for the HTML output
https://storybook.js.org/addons/@whitespace/storybook-addon-html

Needs more tweaking but we can do that later.

Reply:
Cool - that HTML tab is really nice